components/sekhras/actions/sekhraEstimation.js

```diff
@@ -0,0 +1,70 @@
+const getRoute = require('../../../web/api/mapQuest/route');
+const logger = require('../../../config/logger');
+const HTTP = require('../../../constants/statusCode');
+const statusTypes = require('../constants/status');
+const Sekhra = require('../sekhra');
+const Customer = require('../../user/customer');
+const Coursier = require('../../user/coursier');
+const userTypes = require('../../user/constants/types');
+const getClosest = require('../helpers/closestCoursier');
+const formatShapePoints = require('../helpers/formatShapePoints');
+const Notification = require('../../notification/notif');
+
+module.exports = estimateSekhra = async (sekhra, io) => {
+    const { from, to } = sekhra;
+    const [fromLat, fromLong] = from;
+    let coursiers;
+    try {
+        // Find the 5 closest actif coursiers
+        coursiers = await findClosestActifCoursier({
```

Comment:
The closest isn't always the best option, using ETA would be a better approach .